schema/publication.schema.json

```diff
@@ -6,7 +6,26 @@
     "properties": {
         "@context": {
             "type": "array",
-            "const" : ["https://schema.org", "https://www.w3.org/ns/pub-context"]
+            "items": [
+	            {
+	            	"type": "string",
+	            	"enum": ["https://schema.org", "https://www.w3.org/ns/pub-context"]
+	            },
+	            {
+	            	"type": "string",
+	            	"enum": ["https://schema.org", "https://www.w3.org/ns/pub-context"]
+	            },
+	            {
```

Comment:
Actually, the pub-context should follow the schema.org one, because it adds some extra features to schema.org properties... better keep to the order

Reply:
Okay, then we should probably make this a requirement in 2.5.